templates/react/packages/contracts/mud.config.ts

```diff
@@ -1,13 +1,15 @@
-import { mudConfig } from "@latticexyz/world/register";
+import { defineWorld } from "@latticexyz/world/config/v2";
 
-export default mudConfig({
+export default defineWorld({
   tables: {
     Tasks: {
-      valueSchema: {
+      schema: {
+        id: "bytes32",
         createdAt: "uint256",
         completedAt: "uint256",
         description: "string",
       },
+      key: ["id"],
```

Comment:
OOC it is possible to use shorthand here? 

Or alternatively you could reduce the diff by keeping the `key` name